common/src/stack/command/stack/commands/list/box/__init__.py

```diff
@@ -11,10 +11,10 @@
 # @rocks@
 
 import stack.commands
-
+from stack.argument_processors.box import BoxArgumentProcessor
 
 class command(stack.commands.list.command,
-	      stack.commands.BoxArgumentProcessor):
+	      BoxArgumentProcessor):
```

Comment:
No need to put this on 2 lines

Reply:
These were all existing like this and I just did a find/replace operation.

Reply:
Don't mean you can't fix up the style